openpype/hosts/traypublisher/plugins/create/create_editorial.py

```diff
@@ -645,8 +660,8 @@ def _get_base_instance_data(
         fps = instance_data["fps"]
         variant_name = instance_data["variant"]
 
-        # basic unique asset name
-        clip_name = os.path.splitext(otio_clip.name)[0].lower()
```

Comment:
Pourquoi ne plus mettre en minuscule? Cela ne vas-t il pas poser des soucis ailleurs ou un autre `.lower()` est potentiellement fait?

Reply:
Parce que à Wizz, on traite tous les asset en majuscule, et que là, ça forçait une mise en minuscule, et donc aucune correspondance de shot ou seq n'etait trouvé
Le mieux est que la personne respecte d'elle même la nomenclature min/MAJ lors de la création de l'edl

Reply:
Ne s'agissant que du clip_name, s'il est bien écrit de base, je trouve cela plus secure que de lui appliquer des modification dans le code sans que le user ne soit mis au courant.
Au moins, s'il y a un soucis avec le nom, cela sera plus clair.
Et je n'ai pas vu d'autre utilisation de cette var qui poserai soucis ailleurs